src/esssans/loki/general.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+"""
+Default parameters, providers and utility functions for the loki workflow.
+"""
+
+
+from ..types import (
+    DetectorPixelShape,
+    Incident,
+    LabFrameTransform,
+    LoadedFileContents,
+    NexusDetectorName,
+    NexusInstrumentPath,
+    NeXusMonitorName,
+    NexusSourceName,
+    RunType,
+    TransformationPath,
+    Transmission,
+)
+
+default_parameters = {
+    NexusInstrumentPath: 'instrument',
```

Comment:
I wouldn't expect this to change much... or we could change the loader to normalize it? Did you have any concrete need for being able to customize this?

Reply:
The sample (env) is outside the instrument. But yes, we can do what you suggest.

Reply:
In the end I made a hard-coded variable and removed it from the default parameters.